### PR TITLE
updated about requirement for RSAT-Powershell

### PR DESCRIPTION
**Pull request by elduddz**
just added a section called ##requirements

as we add required modules this will need to be updated to possibly include that or explain how to make sure you have them locally

### CODE REVIEW

**Comment by elduddz** [comment]
just realised I need this section for the colourInjection branch

File: README.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ The module has two stages "discovery" and "apply". During the "discovery" stage
 
 The configuration object should be stored within a file, then before invoking the module, read the file will be read and pass the resulting the object into the module call. This should have the effect of being able to store our active directory configuration within Git.
 
+## Requirements
+
+### RSAT-Powershell Windows Feature
+
+The UKHO.ADM module sits on top of the RSAT-Powershell functions, unfortunately this cannot be added as a Required Module as it is specific to windows 10 and server 2016.
```
Comment (MarcDenman) [inline README.md]: Sorry. Can we use this instead:

The UKHO.ADM module uses RSAT-PowerShell functions. These functions are only available after installing [RSAT](https://support.microsoft.com/en-gb/help/2693643/remote-server-administration-tools-rsat-for-windows-operating-systems) and cannot be downloaded from PsGallery.

#### Install RSAT on a Windows Server 

```powershell
Add-WindowsFeature RSAT-AD-PowerShell 
```

#### Install RSAT on a Windows Desktop

Install relevant version for your [Windows Desktop version](https://support.microsoft.com/en-gb/help/2693643/remote-server-administration-tools-rsat-for-windows-operating-systems).

**Review by MarcDenman — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (elduddz) [reply README.md]: updated based on suggestions

File: README.md [hunk 2]
```diff
@@ -6,6 +6,34 @@ The module has two stages "discovery" and "apply". During the "discovery" stage
...
+Add-WindowsFeature RSAT-AD-PowerShell
+```
+
+#### Install RSAT on a Windows Desktop
+
+Install relevant version for your [Windows Desktop version](https://support.microsoft.com/en-gb/help/2693643/remote-server-administration-tools-rsat-for-windows-operating-systems).
+
+### PSWriteColor (Required Module)
+
+When trying to test this locally you will not be able to load the ADM module unless you also have PSWriteColor (version 0.83)
```
Comment (MarcDenman) [inline README.md]: Can we have this instead please:

```
### PSWriteColor

Local development requires PSWriteColor (version 0.83+) to be installed. When ADM is installed as a module(e.g. deploying on a server) PSWriteColor is installed automatically.

#### Install PSWriteColor

Run `./build/Import-RequiredModules.ps1` to install and import all required modules. 

Alternatively manually install PSWriteColor:

```powershell
Install-Module PSWriteColor -RequiredVersion 0.83
```
```

**Review by MarcDenman — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (elduddz) [reply README.md]: added

**Review by MarcDenman — APPROVED** [review]
LGTM

**Pull request merged**